.pullapprove.yml

```
@@ -1081,7 +1081,9 @@ groups:
       # Groups which are found to have matching conditions are `active`
       # according to PullApprove. If no groups are matched and considered
       # active, we still want to have a review occur.
-      - len(groups.active) == 0
-    reviewers:
-      users:
-        - IgorMinar
+      #
+      # Since our global and global docs approval groups have no conditions
+      # it is set to match all changes, the groups.active value always will
+      # be at least 2, we should consider this matched if 2 or less groups
+      # are active.
```

this isn't clear to me... I'll try to correct.. please review and accept/reword further..

```suggestion
      # Since `global-approvers` and `global-docs-approvers` groups have no `conditions`,
      # these groups will match changes to any files and become `active`.
      # The `groups.active` value will therefore always be at least 2 for any PR that matches
      # the "fallback" group.
      # In order for us to recognize that a PR has no requested reviewer, we should
      # check if the PR has at least 2 active reviewer groups.
```